app/views/editorial/submissions/_content.haml

```diff
@@ -34,10 +34,4 @@
         %div.content-block
           != markdown_content content[:content_body]
 
-    - content.optional_keys_in_order.each do |key|
-      %dt= key.to_s.titleize
-      %dd
-        - if content[key].nil?
-          none
-        - else
-          = content[key]
+    = render partial: "editorial/submissions/node_types/#{node_type_template}", locals: { content: content }
```

Comment:
I disagree - it's simpler as it is. Everything follows the pattern. No exceptions.